test/distributions/test_constraints.py

```diff
@@ -41,6 +48,15 @@ def build_constraint(constraint_fn, args, is_cuda=False):
     t = torch.cuda.DoubleTensor if is_cuda else torch.DoubleTensor
     return constraint_fn(*(t(x) if isinstance(x, list) else x for x in args))
 
+@pytest.mark.parametrize('constraint_fn, result, value', [(e[0], e[1], e[2]) for e in EXAMPLES])
```

Comment:
nit: why not just `@pytest.mark.parametrize('constraint_fn, result, value', EXAMPLES)`?

Reply:
You are right. Thanks for checking!